corehq/apps/analytics/tasks.py

```diff
@@ -338,8 +340,7 @@ def submit_data_to_hub_and_kiss(submit_json):
         try:
             dispatcher(submit_json)
         except Exception, e:
-            logger.error(error_message)
-            logger.exception(e)
+            notify_exception(None, "{msg}: {exc}".format(msg=error_message, exc=str(e)))
```

Comment:
i don't think you need the `str` since `.format` takes care of that. also you might want to make the format string unicode in case there are special characters in the error or exception